outputs.tf

```diff
@@ -18,11 +18,20 @@ output "vision_url" {
   description = "IBM Visual Insights URL"
 }
 
+output "vision_username" {
+  value = "admin"
+  description = "IBM Visual Insights username to login"
+}
 output "vision_password" {
   value = "${random_password.vision_password.result}"
   description = "IBM Visual Insights 'admin' password"
```

Comment:
This line should've changed. in theory we might not supply the 'admin' user in the future. It should read `IBM Visual Insights password`